packages/fxa-payments-server/src/components/fields.tsx

```diff
@@ -82,12 +82,12 @@ export const Field = ({
     <div className={className}>
       <label>
         {label && <span className="label-text">{label}</span>}
-        {children}
         {tooltip && tooltipParentRef && validator.getError(name) && (
           <Tooltip parentRef={tooltipParentRef}>
             {validator.getError(name)}
           </Tooltip>
         )}
+        {children}
```

Comment:
flipping this order here actually buys us some sibling selector CSS from the payment server (`aside + input`) without changing the page layout